CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## 1.0.0-beta.9 (Unreleased)
 ### New Features
+**Azure Communication Calling**
+ - Introduction of  a new struct `CommunicationIdentifierModel` to repesent a union type that is either a `communicationUser`, `phoneNumber`, or `microsoftTeamsUser`.  Along side this new object we introduced `CommunicationIdentifierSerializer` to serialize and deserialize valid JSON. 
```

Comment:
The second sentence isn't necessary, particularly if you don't envision customers using the serializer directly.

Reply:
Agree, Serializer are not to be used by the developers. If Autorest would support, we should have even make the `CommunicationIdentifierModel` and it is nested types internal.

Reply:
Yes, but if they were internal I don't think you'd be allowed to expose them on a public object, so I'm not sure Autorest is the problem here. I think CommunicationIdentifier will have to be public.